ci-operator/config/kubevirt/must-gather/kubevirt-must-gather-master.yaml

```diff
@@ -36,6 +36,9 @@ tests:
         cd hyperconverged-cluster-operator
         sed -i "s#15m#30m#g" ./deploy/kustomize/deploy_kustomize.sh
         CONTENT_ONLY=false MARKETPLACE_MODE=false ./deploy/kustomize/deploy_kustomize.sh
+        oc apply -f https://raw.githubusercontent.com/kubevirt/kubevirt.github.io/master/labs/manifests/vm.yaml
+        oc patch virtualmachine myvm --type merge -p '{"spec":{"running":true}}'
+        oc apply -f https://raw.githubusercontent.com/kubevirt/containerized-data-importer/master/manifests/example/blank-image-datavolume.yaml
```

Comment:
does the test cluster configured with a storage class, to bind the DV into an actual backing storage?